app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -91,6 +91,33 @@ open class MainActivity : AppCompatActivity(),
         private val UNDERLINE = "<u style=\"color:lime\">Underline</u><br>"
         private val STRIKETHROUGH = "<s style=\"color:#ff666666\" class=\"test\">Strikethrough</s><br>" // <s> or <strike> or <del>
         private val ORDERED = "<ol style=\"color:green\"><li>Ordered</li><li>should have color</li></ol>"
+        private val ORDERED_WITH_START = "<h4>Start in 10 List:</h4>" +
+                "<ol start=\"10\">\n" +
+                "    <li>Ten</li>\n" +
+                "    <li>Eleven</li>\n" +
+                "    <li>Twelve</li>\n" +
+                "</ol>"
+        private val ORDERED_REVERSED = "<h4>Reversed List:</h4>" +
+                "<ol reversed>\n" +
+                "    <li>Three</li>\n" +
+                "    <li>Two</li>\n" +
+                "    <li>One</li>\n" +
+                "</ol>"
+        private val ORDERED_REVERSED_WITH_START = "<h4>Reversed Start in 10 List:</h4>" +
+                "<ol reversed start=\"10\">\n" +
+                "    <li>Ten</li>\n" +
+                "    <li>Nine</li>\n" +
+                "    <li>Eight</li>\n" +
+                "</ol>"
+        private val ORDERED_REVERSED_NEGATIVE_WITH_START = "<h4>Reversed Start in 10 List:</h4>" +
+                "<ol reversed start=\"1\">\n" +
+                "    <li>Ten</li>\n" +
+                "    <li>Nine</li>\n" +
+                "    <li>Eight</li>\n" +
+                "</ol>"
+        private val ORDERED_REVERSED_WITH_START_IDENT = "<h4>Reversed Start in 4 List:</h4>" +
+                "<ol reversed>" +
+                "<li>dkfdskfdf\\sd\\</li><li>dfds</li><li>fdf</li><li>dfd</li><li>f</li><li>dsfs<ol><li>fdfd</li><li>f</li><li>dsf</li><li>ds</li><li>d</li><li>g</li><li>sdg</li><li>ds</li></ol></li></ol>\n"
```

Comment:
Looks good but can we make this a bit prettier 😅

Reply:
ahahahaha! 
you are 100% right!